controllers/constants/config.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+import "flag"
+
+var (
+	CreateTreeDeadline = flag.Lookup("create-tree-deadline").Value.(flag.Getter).Get().(int64)
```

Comment:
You do not need to use Lookup function of `flag` package. The value will be automatically updated by `flag` when it is parsed.
```suggestion
	CreateTreeDeadline int64 = 1200
```

Reply:
This will removed coupling to `flag` package in shared libraries whil will make easier development of unit test.